netbox/ipam/models/vlans.py

```diff
@@ -273,3 +275,68 @@ def get_vminterfaces(self):
     @property
     def l2vpn_termination(self):
         return self.l2vpn_terminations.first()
+
+
+class VLANTranslationPolicy(PrimaryModel):
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=100,
+        unique=True,
+    )
+
+    class Meta:
+        verbose_name = _('VLAN translation policy')
+        verbose_name_plural = _('VLAN translation policies')
+        ordering = ('name',)
+
+    def __str__(self):
+        return self.name
+
+
+class VLANTranslationRule(NetBoxModel):
```

Comment:
We probably want to include a `description` field on this model, to allow users to record notes about individual rules.